### ISSUE TITLE
java.nio.charset.UnsupportedCharsetException: cp720

### ISSUE THREAD

**Issue by mousavi007** [keycloak/keycloak#11976]
### Describe the bug

when run keycloak on windows 10, get this error
```
D:\app\keycloak-18.0.0>bin\kc.bat start-dev
java.nio.charset.UnsupportedCharsetException: cp720
        at java.base/java.nio.charset.Charset.forName(Charset.java:529)
        at picocli.CommandLine.charsetForName(CommandLine.java:14607)
        at picocli.CommandLine.getStdoutEncoding(CommandLine.java:14599)
        at picocli.CommandLine.getOut(CommandLine.java:1209)
        at picocli.CommandLine.executeHelpRequest(CommandLine.java:1915)
        at picocli.CommandLine.executeHelpRequest(CommandLine.java:1909)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2176)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at org.keycloak.quarkus.runtime.cli.Picocli.runReAugmentation(Picocli.java:190)
        at org.keycloak.quarkus.runtime.cli.Picocli.runReAugmentationIfNeeded(Picocli.java:112)
        at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:83)
        at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:77)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:60)
        at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:31)
java.nio.charset.UnsupportedCharsetException: cp720
        at java.base/java.nio.charset.Charset.forName(Charset.java:529)
        at picocli.CommandLine.charsetForName(CommandLine.java:14607)
        at picocli.CommandLine.getStdoutEncoding(CommandLine.java:14599)
        at picocli.CommandLine.getOut(CommandLine.java:1209)
        at picocli.CommandLine.executeHelpRequest(CommandLine.java:1915)
        at picocli.CommandLine.executeHelpRequest(CommandLine.java:1909)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2176)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:88)
        at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:77)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:60)
        at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:31)
```

### Version

18.0.0

**Comment by DGuhr** [comment]
Hi @mousavi007 
The issue was solved by #11805 and is targeted for the next release. In the meantime, you can try it out using our nightly zip builds ( I assume you are not using docker, for docker it would be the tag :nightly), those can be found here: https://github.com/keycloak/keycloak/releases/tag/nightly - I will close  here, but feel free to re-open when you still experience problems. :)

**Comment by mousavi007** [comment]
> Hi @mousavi007 The issue was solved by #11805 and is targeted for the next release. In the meantime, you can try it out using our nightly zip builds ( I assume you are not using docker, for docker it would be the tag :nightly), those can be found here: https://github.com/keycloak/keycloak/releases/tag/nightly - I will close here, but feel free to re-open when you still experience problems. :)

problem not solved yet
